### PR TITLE
[Translate ru] billions -> index

### PR DESCRIPTION
**Pull request by DmitrySkripunov**
[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/ru/review/2078)

### CODE REVIEW

File: src/content/ru/billions/_index.yaml [hunk 1]
```diff
@@ -0,0 +1,149 @@
+---
+book_path: "/web/_book.yaml"
+landing_page:
+  custom_css_path: "/web/styles/wf-root.css"
+  header:
+    description: 'Следующий миллиард людей, пришедших в онлайн, будет разнообразным
+      во всех отношениях: локация, культура, уровень компьютерной грамотности, доступ
```
Comment (alipattsev) [inline src/content/ru/billions/_index.yaml]: suggest changing локация to место проживания

**Comment by agektmr** [comment]
Thanks @alipattsev for reviewing the Russian translation!
@petele can you please take look?

File: src/content/ru/billions/_index.yaml [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        label: Проектируйте для оффлайн
+        path: "/web/fundamentals/codelabs/offline/"
+      - classname: wf-small-link
+        label: Уменьшите число запросов ресурсов
+        path: "/web/fundamentals/performance/optimizing-content-efficiency/eliminate-downloads"
+      - classname: wf-small-link
+        label: Сделайте установки и обновления настолько маленькими и быстрыми, насколько
+          это возможно
+        path: "/web/fundamentals/performance/optimizing-content-efficiency/optimize-encoding-and-transfer"
+      description: |-
```
Comment (petele) [inline src/content/ru/billions/_index.yaml]: Please use same style as other YAML files

File: src/content/ru/billions/_index.yaml [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        Низкая пропускная способность и неустойчивое соединение – норма для многих людей, подключающихся к вебу.
+        Большинство все еще выходят онлайн на 2G или \"3G\", которое деградирует до 2G. Даже к 2020 г. более 10% пользователей все еще будут на 2G. Множество смартфонов используются оффлайн, и пользователи регулярно включают режим \"в самолете\", чтобы не потреблять данные.
```
Comment (petele) [inline src/content/ru/billions/_index.yaml]: Lines should break at 80 chars

Comment (petele) [inline src/content/ru/billions/_index.yaml]: Quotes do not need to be escaped

File: src/content/ru/billions/_index.yaml [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+      image_path: images/connectivity.png
+    - buttons:
+      - classname: wf-small-link
+        label: Поймите стоимость загрузки страницы и компонентов приложения
+        path: "/web/fundamentals/design-and-ux/responsive/content#weight"
+      - classname: wf-small-link
+        label: Сократите объем данных, необходимых для взаимодействия
+        path: "/web/fundamentals/performance/optimizing-content-efficiency/"
+      - classname: wf-small-link
+        label: Оптимизируйте навигацию: помогите пользователям получить то, что они хотят
```
Comment (petele) [inline src/content/ru/billions/_index.yaml]: If a value contains a `:`, it needs to be quoted
eg: `label: "Оптимизируйте навигацию: помогите пользователям получить то, что они хотят"`

File: src/content/ru/billions/_index.yaml [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+      - classname: wf-small-link
+        label: Оптимизация транзакций
+        path: "/web/fundamentals/design-and-ux/principles/#commerce-and-conversion"
+      - classname: wf-small-link
+        label: Оптимизация форм
+        path: "/web/fundamentals/design-and-ux/input/forms/"
+      - classname: wf-small-link
+        label: Проектируйте удобные в ориентации макеты для небольших экранов
+        path: "/web/fundamentals/design-and-ux/principles/#usability-and-form-factor"
+      description: Платежные системы недоступны для многих людей в вебе. У многих
```
Comment (petele) [inline src/content/ru/billions/_index.yaml]: Please do not insert escape characters such as `\n`

File: src/content/ru/billions/_index.yaml [hunk 6]
```diff
@@ -0,0 +1,147 @@
+---
+book_path: "/web/_book.yaml"
```
Comment (petele) [inline src/content/ru/billions/_index.yaml]: Please maintain the order and spacing of elements in the doc

**Pull request merged**